guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1) Can consider removing extra pipe symbols and spaces before and after the pipe is not constant.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p><strong>Note:&nbsp;</strong>It seems like there is a formatting difference in the review notebook and here in the comments. Can Ignore the above-mentioned comment.</p><p> </p>